.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
   - "3.5"
   - "3.5.3"
   - "3.5.2"
+  - "3.5.1"
```

Comment:
You need to select an older distro for this build (try `trusty` or `xenial`).

Reply:
See e.g. https://github.com/python/mypy/blob/master/.travis.yml#L29

Reply:
done.